### PR TITLE
Only send 1 event per prediction & fix tests

### PR DESCRIPTION
**Pull request by hmstepanek**
# Overview
1. This changes from sending 1 event per feature/label value to 1 event per prediction (Also referred to as a change in "schema").
    * Replace feature_name & feature_value and label_name & label_value to `feature.<name> = <value>` and `label.<name> = <value>`.
    * Remove feature_type and label_type.
    * Add a new schema attribute.
1. Remove pypy38 from test suite and remove compile dependencies from Dockerfile. Scipy wasn't able to compile consistently regardless of installing the necessary dependencies and adding these extra dependencies broke the kafka compile.

### CODE REVIEW

**Comment by TimPansino** [comment]
To be clear, we're removing PyPy3.8 testing because we can't get scipy to compile, yes? And these dependencies that are being dropped were added for that?

**Comment by TimPansino** [comment]
Ah I didn't read the second half of the PR description, I see you already covered that

File: newrelic/hooks/mlmodel_sklearn.py [hunk 1]
```diff
@@ -283,15 +235,38 @@ def bind_predict(X, *args, **kwargs):
...
     model_name = getattr(instance, "_nr_wrapped_name", class_)
     model_version = getattr(instance, "_nr_wrapped_version", "0.0.0")
     user_provided_feature_names = getattr(instance, "_nr_wrapped_feature_names", None)
+    label_names = getattr(instance, "_nr_wrapped_label_names", None)
     settings = transaction.settings if transaction.settings is not None else global_settings()
 
+    labels = []
+    if return_val is not None:
+        if not hasattr(return_val, "__iter__"):
+            labels = np.array([return_val])
```
Comment (TimPansino) [inline newrelic/hooks/mlmodel_sklearn.py:251]: Have we tested that this works for string return vals? Seems like it might split something like `"abc"` to `["a", "b", "c"]`.

**Comment by hmstepanek** [comment]
It has been validated in the UI. I'm checking on that string related question now...

File: newrelic/hooks/mlmodel_sklearn.py (hunk 1 above)
Comment (hmstepanek) [reply newrelic/hooks/mlmodel_sklearn.py:251]: Very good point! For the label columns, the values are only numeric (these values come from the output of the model prediction) so we should not need to handle this scenario.

**Review by TimPansino — APPROVED** [review]
